src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -49,10 +49,6 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        if (!$container->getParameter('kernel.debug') || !$container->has('logger')) {
-            $container->removeDefinition('messenger.middleware.debug.logging');
-        }
```

Comment:
I needed move this part to `FrameworkExtension` to make tests pass ([see old failures](https://travis-ci.org/symfony/symfony/jobs/365937393#L2677))